Fix project name in Installation Instructions

e28eta opened the pull request:
Replaces "SwiftCheck" with "Concurrent" in the Swift Package Manager Installation instructions.

e28eta committed:
Fix project name in Installation Instructions

Replaces "SwiftCheck" with "Concurrent" in the Swift Package Manager Installation instructions.

CodaFi commented:
Hilarious how long this has been there, great catch.

Could you also update the Swift PM declarations to Swift 4’s format while you’re in the area?

e28eta committed:
Update Swift PM dependency declaration to Swift 4

Per documentation at https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md#dependencies

I haven't actually used Swift PM, and I'm on iPad so I wasn't able to verify this is correct, but I believe it matches the documentation.

CodaFi commented:
⛵️
